packages/sourcegraph-extension-api/src/sourcegraph.d.ts

```diff
@@ -724,9 +724,16 @@ declare module 'sourcegraph' {
 
         /**
          * An event that is fired when a new text document is opened.
+         *
+         * @deprecated this is deprecated and will be removed soon. Use {@link workspace.textDocumentAdds} instead.
```

Comment:
```suggestion
         * @deprecated Renamed to {@link workspace.textDocumentAdds}.
```

So that it is clear that there is no behaviour change.